tests/runmode_test/test.yaml

```diff
@@ -0,0 +1,10 @@
+requires:
+  min-version: 7.0.0
+  pcap: false
+
+checks:
+  - shell:
+      args: suricata
+      expect: 1  # Expect an exit code of 1, indicating an error
```

Comment:
When you want to indicate that a test expects Suricata to exit with error code 1, you should add an `exit-code: 1` line, outside the checks. See:
https://github.com/OISF/suricata-verify/blob/master/tests/detect-strip_whitespace-01/test.yaml#L5C1-L5C1